source/docs/yearly-overview/yearly-changelog.rst

```diff
@@ -0,0 +1,138 @@
+
+- Added a ``simulationPeriodic()`` method to Subsystem. This method runs periodically during simulation, in addition to the regular ``periodic()`` method.
+
+General Library
+^^^^^^^^^^^^^^^
+- Added a ``SpeedControllerGroup`` constructor that takes a ``std::vector<>`` (C++) / ``SpeedController[]`` (Java), allowing the list to be constructed dynamically.
+
+- Added methods (``isOperatorControlEnabled()`` and ``isAutonomousEnabled()``) to check game and enabled state together.
+
+- Added a ``ScopedTracer`` class for C++ teams to be able to time pieces of code. Simply instantiate the ``ScopedTracer`` at the top of a block of code and the time will be printed to the console when the instance goes out of scope.
+
+- Added a static method ``fromHSV(int h, int s, int v)`` to create a ``Color`` instance from HSV values.
+
+- Added RT priority constructor to ``Notifier`` in C++. This makes the thread backing the Notifier run at real-time priority, reducing timing jitter.
+
+- Added ``EllipticalRegionConstraint``, ``RectangularRegionConstraint``, and ``MaxVelocityConstraint`` to allow constraining trajectory velocity in a certain region of the field.
+
+- Added equals() operator to the ``Trajectory`` class to compare two or more trajectories.
+
+- Added zero-arg constructor to the ``Trajectory`` class in Java that creates an empty trajectory.
+
+- Added a special exception to catch trajectory constraint misbehavior. This notifies users when user-defined constraints are misbehaving (i.e. min acceleration is greater than max acceleration).
+
+- Added a ``getRotation2d()`` method to the ``Gyro`` interface. This method automatically takes care of converting from gyro conventions to geometry conventions.
+
+- Added angular acceleration units for C++ teams. These are available in the ``<units/angular_acceleration.h>`` header.
+
+- Added X and Y component getters in ``Pose2d`` - ``getX()`` and ``getY()`` in Java, ``X()`` and ``Y()`` in C++.
+
+- Added implicit conversion from ``degree_t`` to ``Rotation2d`` in C++. This allows teams to use a degree value (i.e. ``47_deg``) wherever a ``Rotation2d`` is required.
+
+- Fixed some spline generation bugs for advanced users who were using control vectors directly.
+
+- Fixed theta controller continuous input in swerve examples. This fixes the behavior where the shortest path is not used during drivetrain rotation.
+
+- Deprecated ``units.h``, use individual units headers instead.
+
+- Added support for scheduling functions more often than the robot loop via addPeriodic() in TimedRobot. Previously, teams had to make a Notifier to run feedback controllers more often than the TimedRobot loop period of 20ms (running TimedRobot more often than this is not advised). Now, users can run feedback controllers more often than the main robot loop, but synchronously with the TimedRobot periodic functions so there aren't any thread safety issues.
+
```

Comment:
What about state-space?

Reply:
It's still WIP, we haven't all gotten around to adding items yet.

Reply:
We are working on this document outside of github.